src/commands/interviews/interviewerSignup.ts

```diff
@@ -37,7 +38,7 @@ class InterviewerSignupCommand extends BaseCommand {
 
     // Add or update interviewer info
     await upsertInterviewer(id, parsedUrl);
-    return message.reply(`your info has been updated. Thanks for helping out!`);
+    return message.reply(`your info has been updated. Thanks for helping out! ` + emoteList['codeyLove'].toString());
```

Comment:
I actually don't know JS standard lol, do yall use template literals? It's more concise with `${emoteList['codeyLove']}`. It doesn't matter too much with end of string case like ^, but if we wanted variables within a sentence, defs easier.

Reply:
yeah agreed, should use template string so you don't need the `+`